src/harness/reference_models/aggregate_interference/test_data/fss_record_1.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "incumbent/ibfs/1",
+  "type": "FSS",
+  "deploymentParam":
+  [
+    {
+      "installationParam": {
+        "latitude": 40.44694,
+        "longitude":  -97.86621,
+        "elevation": 100,
+        "height": 26.5,
+        "heightType": "AGL",
+        "antennaGain": 16,
+        "antennaBeamwidth": 30,
+        "antennaAzimuth": 180,
+        "antennaDowntilt": -5,
+        "antennaElevationAngle":40
+      },
+      "operationParam": {
+        "operationFrequencyRange": {
+          "lowFrequency": 3700000000,
+          "highFrequency": 4200000000
+        }
+      },
+      "weight1": 0.5,
+      "weight2": 0.5,
+      "ttc": true
```

Comment:
ttc should be up one level. See for referebnce the #486

Reply:
#486 just merged in, will sync up in next drop.